src/generic_chunk.c

```diff
@@ -27,6 +24,12 @@ static ChunkFuncs regChunk = {
     .LoadFromRDB = Uncompressed_LoadFromRDB,
 };
 
+ChunkIterFuncs uncompressedChunkIteratorClass = {
+    .Free = Uncompressed_FreeChunkIterator,
+    .GetNext = Uncompressed_ChunkIteratorGetNext,
+    .GetPrev = Uncompressed_ChunkIteratorGetPrev,
+};
```

Comment:
PR #508 moves the function pointers from the `Series struct` to the `Chunk struct`. This will make this struct and passing it to `NewChunkIterator` unnecessary as the chunk already owns a pointer to these functions.